packages/global-styles/src/hooks/use-get-global-styles-base-config.ts

```diff
@@ -3,7 +3,7 @@ import wpcomRequest from 'wpcom-proxy-request';
 import type { GlobalStylesObject } from '../types';
 
 const useGetGlobalStylesBaseConfig = ( siteId: number | string, stylesheet: string ) => {
-	return useQuery< any, unknown, unknown >(
+	return useQuery(
```

Comment:
This was preventing type inference from working -- the type parameter to `wpcomRequest` below was actually enough to get the return type inferred correctly. (But that didn't work with any/unknown type params to useQuery)